docs/topics/wasm/wasm-overview.md

```diff
@@ -1,61 +1,74 @@
 [//]: # (title: Kotlin Wasm)
 
-> Kotlin Wasm is [Experimental](components-stability.md).
-> It may be changed at any time. Use it only for evaluation purposes.
+> Kotlin Wasm is [Alpha](components-stability.md).
+> It may be changed at any time. You can use it in scenarios before production. We would appreciate your feedback on it in [YouTrack](https://kotl.in/issue).
```

Comment:
> before production

Is it ok? In other places, we use `pre-production`.

Reply:
Hm, we don't use pre-production in our docs at all.
Usually we say "before using in production", so it seems OK for me.